index.ts

```diff
@@ -7,5 +8,5 @@ app.get("/*", (req, res) => {
 });
 
 app.listen(port, () => {
-  "Server is listening...";
+  console.log("Server is listening...");
```

Comment:
How do I know that my Node application is connected to the database?

Reply:
What I want here is log that says database is connected and it's running.

Reply:
> How do I know that my Node application is connected to the database?

We don't. [Prisma documentation](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/connection-management) says: _PrismaClient automatically connects when you run your first query, creates a connection pool, and disconnects when the Node.js process ends._

Also, it says: _It is not necessary to call $connect() thanks to the lazy connect behavior: The PrismaClient instance connects lazily when the first request is made to the API ($connect() is called for you under the hood)._

Would it be enough to just log the text "Database started" (or something similar), or even to include the database URL (from .env file)?

Reply:
> What I want here is log that says database is connected and it's running.

One more thing about this. Since `start_db.sh` starts the database, should the message "Database started" be printed from this file? That way, if the database doesn't start for some reason, that message won't be printed.

Message that app is running would still be printed from `index.ts`.

Reply:
A good practice is to have development and production setup as close as possible, because it's easier to debug when some exceptions happen in production environment. In this case, it's development setup and it would be fine to leave Prisma to do the lazy connection. But on production I want it to fail fast, meaning we don't want to wait someone to create first request and then discover that connection with database failed. 

So it would be nice to [connect database explicitly](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/connection-management#calling-connect-explicitly) on both development and production as well. So that when we start the service we see immediately if connection to the database is successful of not.